php/src/app/Repo/PostRepo.php

```diff
@@ -22,4 +22,24 @@ public static function get_posts(\PDO $pdo): array
           ),
         $stmt->fetchAll(\PDO::FETCH_ASSOC));
     }
+
+    /**
+     * @param \PDO $pdo
+     * @param string $title
+     * @param string $content
+     * @param string $thumnail_url
+     * @return void
+     */
+    public static function create_posts(\PDO $pdo, string $title, string $content, string $thumnail_url)
```

Comment:
slackでも話した通り、Modelを受け取るようにしたいです
```suggestion
    public static function create_posts(\PDO $pdo, Post $post)
```